misk-aws/src/main/kotlin/misk/jobqueue/sqs/SqsJobConsumer.kt

```diff
@@ -107,6 +101,27 @@ internal class SqsJobConsumer @Inject internal constructor(
       }.join()
     }
 
+    /**
+     * List containing arbitrary numbers. The size of the list is the number of receivers to use.
+     */
+    private fun receivers(): List<Int> {
+      val numReceiversForQueue = receiversForQueue()
+      val shouldUsePerQueueConfig = numReceiversForQueue >= 0
+      if (shouldUsePerQueueConfig) {
```

Comment:
I think you should change the order of the check. If you set `POD_CONSUMERS_PER_QUEUE > 0`, switch over to your new setting. Otherwise you need to change both flags to toggle this on. If both are 0, then queues are shut off